transformer/transformers/ampruntimejs_test.go

```diff
@@ -0,0 +1,70 @@
+package transformers_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ampproject/amppackager/transformer/internal/amphtml"
+	tt "github.com/ampproject/amppackager/transformer/internal/testing"
+	"github.com/ampproject/amppackager/transformer/transformers"
+	"golang.org/x/net/html"
+)
+
+func TestStripJS(t *testing.T) {
+	tcs := []tt.TestCase{
+		{
+			Desc:     "no script node",
+			Input:    "<head></head><body></body>",
+			Expected: "<head></head><body></body>",
+		},
+		{
+			Desc:     "no prefix",
+			Input:    "<script src=main.js/>",
+			Expected: "<script src=main.js/>",
+		},
+		{
+			Desc:     "no suffix",
+			Input:    "<script src=\"https://cdn.ampproject.org\"/>",
+			Expected: "<script src=\"https://cdn.ampproject.org\"/>",
+		},
+		{
+			Desc:     "transformation",
+			Input:    "<script async src=\"https://cdn.ampproject.org/v0.js\"></script>",
+			Expected: "<script async src=\"https://cdn.ampproject.org/v0.sxg.js\"></script>",

```

Comment:
I don't think there's a hard and fast rule that forces signed exchanged files to have the sxg file extension, but conceptually when I look at v0.sxg.js, I'm inclined to assume it's a Javascript file, not a Signed exchange file. In my opinion, that's misleading but I defer to @twifkak on whether or not this is acceptable.

Reply:
The intent is to mimic what we have done for module builds, which had the file extension `.mjs`.